### ISSUE TITLE
Getting score of multin label task

### ISSUE THREAD

**Issue by xx257xx** [open-mmlab/mmclassification#475]
### Checklist
- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

I’m working on multi label task.
Is there simple way to get score of each label?

My present code is as follows

    data = test_pipeline(data)
    data = collate([data], samples_per_gpu=1)
    if next(model.parameters()).is_cuda:
        # scatter to specified GPU
        data = scatter(data, [device])[0]

    # forward the model
    with torch.no_grad():
        scores = model(return_loss=False, **data)
        pred_score = np.max(scores, axis=1)[0]
        pred_label = np.argmax(scores, axis=1)[0]
        result = {'pred_label': pred_label, 'pred_score': float(pred_score), 'scores' : scores}
    result['pred_class'] = model.CLASSES[result['pred_label']]
    return result['scores']

### Post related information
1. The output of `pip list | grep "mmcv\|mmcls\|^torch"`
[here]
2. Your config file if you modified it or created a new one.
```python
[here]
```
3. Your train log file if you meet the problem during training.
[here]
4. Other code you modified in the `mmcls` folder.
[here]

**Comment by mzr1996** [comment]
Hello, I think you can simply specify `--out` in `test.py`, which refers to [the docs](https://mmclassification.readthedocs.io/en/latest/getting_started.html#inference-and-test-a-dataset).
The output file will record all inference results like scores of each class.
